Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -293,6 +302,22 @@ public virtual bool CanLaunchWebBrowser
             ApplicationUserAgent = Name.Replace(' ', '-') + "/" + ApplicationVersionString;
         }
 
+        /// <summary>
+        /// Temporary function to migration network settings out of system.xml and into network.xml.
+        /// TODO: remove at the point when a fixed migration path has been decided upon.
+        /// </summary>
+        private void MigrateNetworkConfiguration()
```

Comment:
Why not just use the migration framework we use for everything else?

Reply:
I did try (hence the wrong namespace above) but the migration framework is done too late in the process. 
Networkmanager is created before everything else.